lib/all_your_base_cases.rb

```diff
@@ -57,16 +57,17 @@ def error_message
 
 class AllYourBaseCase::PreProcessor
   class << self
+    attr_reader :row
+
     def call(row)
       @row = row
 
       row.merge('expected' => expected_value)
     end
 
+    private :row
     private
 
```

Comment:
The warning was removed in ruby 2.3, so in all later versions it isn't there and you won't see it in 2.4. Since we use 2.1.2, it shows up in CI and whatnot.

I'm not sure if moving the reader under private but keeping the explicit private with method name will still make a warning. I'll give this a try when I'm home.

Sorry for the lack of code samples... aparently the iOS keyboard doesn't have a back tick key? 😞